features/log-analyzer/org.wso2.carbon.analytics.shared.la.apim.feature/src/main/resources/gadgets/LogAPIMArtifactDeleted/js/main.js

```diff
@@ -50,78 +50,95 @@ function initialize() {
-        '<p>Please select a date range to view stats.</p>' +
-        '</div>' +
-        '</div>';
-};
-
-function getEmptyRecordsText() {
-    return '<div class="status-message">' +
-        '<div class="message message-info">' +
-        '<h4><i class="icon fw fw-info"></i>No records found</h4>' +
-        '<p>Please select a date range to view stats.</p>' +
-        '</div>' +
-        '</div>';
-}
-
 $(document).ready(function () {
     initialize();
+    nanoScrollerSelector.nanoScroller();
 });
 
 function fetch() {
-    dataM.length = 0;
+    receivedData.length = 0;
     var queryInfo;
-    queryInfo = {
+    var queryForSearchCount = {
         tableName: "LOGANALYZER_APIM_ARTIFACT_DELETED_DAILY",
         searchParams: {
-            query: "_timestamp: [" + from + " TO " + to + "]",
-            start: 0, //starting index of the matching record set
-            count: 100 //page size for pagination
+            query: "_timestamp: [" + timeFrom + " TO " + timeTo + "]",
         }
     };
-    client.search(queryInfo, function (d) {
-        var obj = JSON.parse(d["message"]);
+
+    client.searchCount(queryForSearchCount, function (d) {
         if (d["status"] === "success") {
-            for (var i = 0; i < obj.length; i++) {
-                dataM.push([obj[i].values.artifact, obj[i].values.artifactCount]);
-            }
-            if (!initState) {
-                redrawLogAPIMArtifactTableChart();
+            var totalRecordCount = d["message"];
+            if (totalRecordCount > 0) {
+                queryInfo = {
+                    tableName: "LOGANALYZER_APIM_ARTIFACT_DELETED_DAILY",
+                    searchParams: {
+                        groupByField: "artifact",
+                        query: "_timestamp: [" + timeFrom + " TO " + timeTo + "]",
+                        aggregateFields: [
+                            {
+                                fields: ["artifactCount"], //Array of field names used as variables for aggregateFunction
+                                aggregate: "SUM", //Aggregate Function Name
+                                alias: "artifactCountSum"   //Alias given to the result after aggregation
+                            }
+                        ],
+                        aggregateLevel: 0,
+                        parentPath: [],
+                        noOfRecords: totalRecordCount
+                    }
+                };
+                client.searchWithAggregates(queryInfo, function (d) {
+                    var obj = JSON.parse(d["message"]);
+                    if (d["status"] === "success") {
+                        for (var i = 0; i < obj.length; i++) {
+                            receivedData.push([obj[i].values.artifact, obj[i].values.artifactCountSum]);
+                        }
+                        drawDeletedArtifactTable();
+                    }
+                }, function (error) {
+                    console.log(error);
```

Comment:
This console logs removed in following commit : https://github.com/wso2/shared-analytics/commit/f1f9a8cf9493cfc0ad276b355d4a1d6144ae7a83